pandas/core/series.py

```diff
@@ -354,15 +354,18 @@ def _init_dict(self, data, index=None, dtype=None):
         # Looking for NaN in dict doesn't work ({np.nan : 1}[float('nan')]
         # raises KeyError), so we iterate the entire dict, and align
         if data:
-            keys, values = zip(*data.items())
-            values = list(values)
+            # GH:34717
+            # Using the below way for generating keys and values
```

Comment:
you don't need these comments; they are confusing because unless you are looking at the diff you don't know what they refer (you can leave the issue ref if you want)